website/addons/osfstorage/model.py

```diff
@@ -218,6 +241,20 @@ def path(self):
     def node(self):
         return self.node_settings.owner
 
+    @property
+    def rented(self):
```

Comment:
Why not just use the `renter` field, then?

At the very least, I would argue that the function's name does not match what it does.